modules/python/Makefile.security

```diff
@@ -0,0 +1,15 @@
+BANDIT=$(call which, bandit)
+BANDIT_FLAGS=--level --level 
```

Comment:
How about `-ll -ii` to reduce false positives?

Reply:
Good idea, added the confidence flags